packages/e2e-tests/specs/experiments/widgets.test.js

```diff
@@ -0,0 +1,363 @@
+/**
```

Comment:
The screen isn't experimental so we can move this file out of `experiments`.

Reply:
I'd probably name this file `widgets-editor.test.js` too.

Reply:
Where should we move this to though? `editor/various`?

Reply:
Hmmm we will probably end up having more than one file so maybe a new `widgets` folder.